### ISSUE TITLE
Тестовая кампания

### ISSUE THREAD

**Issue by EvgenLozov** [user50/traffic-cost-optimization#2]
Еще возникла идея одной фишки по автоматизации:
1. сделать выбор связки тестовой и вайт компании.
2. В тестовой компании мы будем тестировать таргеты и те таргеты, которые не дали конверсии на определенном кол-ве редиректов блокируем.
3. Те, что дали конверсии, блокируем в тестовой компании и добавляем в вайт-компанию, назначая максимальный бид по логике расчета, описанной ранее.

Соответственно, в тестовой компании нет нужды управлять бидом таргетов, только блокировка и перенос в вайт-компанию.
В вайт-компании уже управляем бидом и контролируем что таргет не перестает приносить конверсии (если перестает - блокируем)

Стоимость 50 уе

**Comment by proyuri** [comment]
Верно. Нужно будет как-то обозначать связки таких тестовых и вайт компаний
